composer.json

```diff
@@ -53,5 +54,8 @@
         }
     },
 
+    "prefer-stable": true,
+    "minimum-stability": "dev",
```

Comment:
Yeah, well, the only challenge you will how is that now you will need to introduce an extension point for extension instantiators, which is definitely a head-scratcher.